README.md

```diff
@@ -42,15 +41,17 @@ host$ docker build -t core .
 host$ docker run -v "$(pwd)":"/code" core npm run build
 ```
 
+Build the core library using `npm run build` and run unit tests with `npm run test`.
```

Comment:
nit - should it say `cd ./core` first?

Reply:
Not for the docker image, as the instructions have you mounting to `/core` folder. This might be misleading for those not using the docker image though, I'll add a little clarification.